staging/src/k8s.io/kubectl/pkg/cmd/create/create.go

```diff
@@ -60,6 +60,7 @@ type CreateOptions struct {
 	Selector         string
 	EditBeforeCreate bool
 	Raw              string
+	Labels           string
```

Comment:
By my comment below, this can be an array ([]string). But I don't know if we want to label things created directly by files (like kubectl create -l mykey=myvalue -f mypod.yaml)